src/Illuminate/Http/Request.php

```diff
@@ -459,7 +459,7 @@ protected function convertUploadedFiles(array $files)
      *
      * @param  string  $key
      * @param  mixed  $default
-     * @return \Symfony\Component\HttpFoundation\File\UploadedFile|array|null
+     * @return \Illuminate\Http\UploadedFile|array|null
```

Comment:
This is targeting 5.2 onward. If you are using 5.1, then you are correct that it returns a Symfony UploadedFile object.